src/main/scala/cromwell/engine/io/gcs/GoogleConfiguration.scala

```diff
@@ -12,6 +12,7 @@ import scalaz._
 sealed trait GoogleCromwellAuthMode
 final case class ServiceAccountMode(accountId: String, pemPath: String) extends GoogleCromwellAuthMode
 final case class UserMode(user: String, secretsFile: String, datastoreDir: String) extends GoogleCromwellAuthMode
+final case object ApplicationDefaultMode extends GoogleCromwellAuthMode
```

Comment:
True, if we really think it's right to do this then we should just tune our IntelliJs.  But to me this feels "belt and suspenders" since `object`s can't be subclassed anyway.